src/english_score_get_data.py

```diff
@@ -0,0 +1,70 @@
+import sys
+import os
+import pandas as pd
+from sklearn.model_selection import train_test_split
+import click
+
+# Add the src directory to the system path
+root_dir = os.path.dirname(os.path.abspath(__file__))  # Get the absolute path of the script
+src_dir = os.path.join(root_dir, 'src')                # Define the src directory path
+sys.path.append(src_dir)                               # Add src directory to the system path
+
+@click.command()
+@click.option("--url", required=True, type=str, help="URL of the dataset to download.")
+@click.option(
+    "--output_folder_path",
+    required=True,
+    type=str,
+    help="Path to the output folder where files will be saved.",
+)
+def main(url, output_folder_path, sample_size=0.3, test_size=0.3):
+    """
+    Downloads, samples, splits, and saves a dataset.
+
+    This function downloads a dataset from a specified URL using pandas.
+    It samples a fraction of the dataset (default: 30%) and saves this sampled dataset
```

Comment:
I believe defaults are being dealt with differently in Nando's script (with `default=` parameter in click option). LGTM otherwise but for consistency we could change this